bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,323 @@
+                                              offset=random.randint(0, 150))
+        except KeyError:
+            await ctx.send_help('games')
+            return
+
+        # Create pages and paginate
+        pages = [await self.create_page(game) for game in games]
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title=f'Random {genre} Games'))
+
+    @games.command(name='top', aliases=['t'])
+    async def top(self, ctx: Context, amount: int = 10) -> None:
+        """
+        Get current Top games in IGDB.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than top 25 games.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 top game.")
+            return
+
+        games = await self.get_games_list(self.http_session, amount, sort='total_rating desc',
+                                          additional_body="where total_rating >= 90; sort total_rating_count desc;")
+
+        pages = await self.get_pages(games)
+        await ImagePaginator.paginate(pages, ctx, Embed(title=f'Top {amount} Games'))
+
+    @games.command(name='genres', aliases=['genre', 'g'])
+    async def genres(self, ctx: Context) -> None:
+        """Get all available genres."""
+        await ctx.send(f"Currently available genres: {', '.join(f'`{genre}`' for genre in self.genres)}")
+
+    @games.command(name='search', aliases=['s'])
+    async def search(self, ctx: Context, *, search: str) -> None:
+        """Find games by name."""
+        lines = await self.search_games(self.http_session, search)
+
+        await LinePaginator.paginate((line for line in lines), ctx, Embed(title=f'Game Search Results: {search}'))
+
+    @games.command(name='company', aliases=['companies'])
+    async def company(self, ctx: Context, amount: int = 5) -> None:
+        """
+        Get random Game Companies companies from IGDB API.
+
+        Support amount parameter. Max is 25, min is 1.
+        """
+        if amount > 25:
+            await ctx.send("You can't get more than 25 companies at once.")
+            return
+        elif amount < 1:
+            await ctx.send("You can't get less than 1 company.")
+            return
+
+        companies = await self.get_companies_list(self.http_session, amount, random.randint(0, 150))
+        pages = [await self.create_company_page(co) for co in companies]
+
+        await ImagePaginator.paginate(pages, ctx, Embed(title='Random Game Companies'))
+
+    async def get_games_list(self,
+                             client: ClientSession,
+                             limit: int,
+                             genre: str = None,
+                             sort: str = None,
+                             additional_body: str = "",
+                             offset: int = 0) \
+            -> List[Dict[str, Any]]:
```

Comment:
The prevalent style for long function signatures is as follows:
```python
def my_function(
    a: type,
    b: type = default
) -> return_type:
```
Please try to avoid line continuation with `\`. It is *very* rarely necessary.

Additionally:
- Does `client` need to be a param? You always pass in `self.http_session`. This function can simply grab the session from `self`.
- I'd prefer using `amount` rather than `limit`, especially if the parameter isn't explained anywhere. When calling the function, it is referred to as `amount`.
- What are allowed values for `sort`? What does it do?
- What am I supposed to pass into `additional_body`?
- What is `offset`?

A docstring of `Get Games List from IGDB API.` may not cut it for this function signature.

Regarding the type annotations - if a param may be of type `T` or `None`, annotate it as `Optional[T]`. Some of these string params default to None, whereas some default to an empty string. It's up to you to choose which one makes more sense: if the query can be safely constructed with empty strings, you can save yourself a conditional. If you'll have to check whether a non-default param was provided, using `None` to denote "no param was passed" in a conditional may look cleaner.